cumulus_etl/formats/batched_files.py

```diff
@@ -47,9 +47,9 @@ def __init__(self, *args, **kwargs) -> None:
             self.root.rm(parent_dir, recursive=True)
         except FileNotFoundError:
             pass
-        self.root.makedirs(parent_dir)
```

Comment:
Unrelated, but I found that we were making folders for resources that didn't get written when running i2b2, because we made each task's folder early. Just moving that to when we write a batch out, so that we don't make so many spurious folders.